apps/arweave/include/ar_config.hrl

```diff
@@ -191,7 +191,11 @@
 	cm_api_secret = not_set,
 	cm_exit_peer = not_set,
 	cm_peers = [],
-	cm_poll_interval = ?DEFAULT_CM_POLL_INTERVAL
+	cm_poll_interval = ?DEFAULT_CM_POLL_INTERVAL,
+	pool_server = false,
+	pool_client = false,
+	pool_host = not_set,
+	pool_api_key = not_set
```

Comment:
The idea is to use a shared secret here? like with the `cm_api_secret`

Reply:
The difference is nodes will connect with the pool over https.

Reply:
Oh awesome.